<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="musicState">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>333</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>   Music Off</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>390</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>391</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </widget>
  <widget class="QLabel" name="volume">
   <property name="geometry">
    <rect>
     <x>576</x>
     <y>392</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>13 Klee</family>
     <pointsize>25</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 25pt bold &quot;Klee&quot;;
	color: rgb(0,0,0);
	background-color: rgb(212, 212, 212);
	border: none;
}

QPushButton:hover{
	background-color: rgb(222, 222, 222);
}
QPushButton:pressed{
	background-color: rgb(180, 180, 180);
}</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
